tests/commonTemplates_test.go

```diff
@@ -208,6 +208,30 @@ var _ = Describe("Common templates", func() {
 	})
 
 	Context("resource change", func() {
+
+		Context("namespace change", func() {
+			AfterEach(func() {
+				strategy.RevertToOriginalSspCr()
+			})
+
+			It("[test_id:6057] should not fail to update commonTemplates.namespace", func() {
+				foundSsp := getSsp()
+				const testNamespace = "ssp-operator-functests"
+				Expect(foundSsp.Spec.CommonTemplates.Namespace).ToNot(Equal(testNamespace), "namespaces should not equal")
+				foundSsp.Spec.CommonTemplates.Namespace = testNamespace
+
+				err := apiClient.Update(ctx, foundSsp)
+				Expect(err).ToNot(HaveOccurred())
+
```

Comment:
The test needs to wait until SSP operator is finished with reconciliation. You can call: `waitUntilDeployed()`